src/auslib/db.py

```diff
@@ -1696,14 +1711,20 @@ def __init__(self, db, metadata, dialect, baseName):
         super(SignoffsTable, self).__init__(db, dialect, versioned=False, historyClass=HistoryTable)
 
     def insert(self, changed_by=None, transaction=None, dryrun=False, **columns):
-        if "sc_id" not in columns or "role" not in columns:
+        if "sc_id" not in columns and "role" not in columns:
```

Comment:
We do we need to update `or` to `and` here?

Reply:
We do not need to update, that was an error on my part